Lib/test/test_pathlib.py

```diff
@@ -575,8 +575,6 @@ def test_with_suffix_common(self):
         self.assertRaises(ValueError, P('a/b').with_suffix, '.c/.d')
         self.assertRaises(ValueError, P('a/b').with_suffix, './.d')
         self.assertRaises(ValueError, P('a/b').with_suffix, '.d/.')
-        self.assertRaises(ValueError, P('a/b').with_suffix,
-                          (self.pathmod.sep, 'd'))
```

Comment:
Thinking out loud a bit here -- So, if I understand correctly, `P('a/b').with_suffix(('\\', 'd'))` fails with `ValueError` on `main`, but fails with `AttributeError` on this PR. Neither seems quite right -- ideally, I feel like it should fail with `TypeError`. _But_, I think probably it's not worth jumping through hoops to implement that. Passing a `tuple` into `Path.with_suffix` would be a pretty strange thing to do :)

Reply:
Yep exactly my thinking too! It should probably be a `TypeError`, but this PR doesn't make it any worse.